tests/test_runs/test_run_functions.py

```diff
@@ -518,12 +518,14 @@ def _run_and_upload_regression(
             sentinel=sentinel,
         )
 
-    def test_run_and_upload_logistic_regression(self):
+    @unittest.mock.patch("warnings.warn")
+    def test_run_and_upload_logistic_regression(self, warn_mock):
         lr = LogisticRegression(solver="lbfgs")
         task_id = self.TEST_SERVER_TASK_SIMPLE[0]
         n_missing_vals = self.TEST_SERVER_TASK_SIMPLE[1]
         n_test_obs = self.TEST_SERVER_TASK_SIMPLE[2]
         self._run_and_upload_classification(lr, task_id, n_missing_vals, n_test_obs, "62501")
+        self.assertLessEqual(warn_mock.call_count, 3)
```

Comment:
Please explain why we expect three warnings.

Reply:
Honestly, I don't understand the 3 myself and would have expected 1. I will simply increase the max_iter so this test passes.

Reply:
Ah, I now understand. We call run_model_on_task once, and then twice a helper function called `_rerun_model_and_compare_predictions`. Anyway, I think increasing the number iterations is a bit cleaner.